src/app/pages/dev/dev.page.ts

```diff
@@ -15,17 +15,15 @@ export class DevPage extends Tab3Page implements OnInit {
   packageOverride: string = "default"
   firebaseToken: string = ""
 
-  ngOnInit() {
+  async ngOnInit() {
```

Comment:
why async?

Reply:
This must have happened on accident. I will revert this change.